### PR TITLE
Fixed validate against release branch

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/29352

### CODE REVIEW

**Comment by Itay4** [comment]
@Shellyber do we have content builds we tested this against?
also, i see this function is not covered at all by unit test or integration test - lets make the effort to test this please

**Review by yaakovi — COMMENTED** [review]
looks ok to me.
following Itay's comment, need to test against the following builds:
* regular build
* feature branch build (checkout from `v4.5.0` branch)
* release branch (branch name `20.13.0` and modify the version in `config.yml` file as well)

**Comment by Shellyber** [comment]
@Itay4 @yaakovi 
Running against release branch: https://app.circleci.com/pipelines/github/demisto/content/33035/workflows/309ab512-ca54-42a1-99db-f6378a0511ac/jobs/144420
running against 4.5.0 branch:  https://app.circleci.com/pipelines/github/demisto/content/33037/workflows/4c84ec09-8baa-4ff9-b472-93785719b186/jobs/144459
running against regular branch: https://app.circleci.com/pipelines/github/demisto/content/33047/workflows/a44db49b-7896-4ee7-b35a-0e2fa5f58327/jobs/144506

File: demisto_sdk/commands/validate/tests/validators_test.py [hunk 1]
```diff
@@ -1025,3 +1025,24 @@ def test_content_release_identifier_exists():
...
+    """
+    Given
+        - Prev_ver to test on.
+    When
+        - Run the add origin command.
+    Then
+        - validate add_origin runs as expected.
+    """
+    validate_manager = ValidateManager()
+    validate_manager.branch_name = 'test-branch'
```
Comment (yaakovi) [inline demisto_sdk/commands/validate/tests/validators_test.py:1046]: add tests for the different types of branches too (master, release branch, v4.5.0)

File: demisto_sdk/commands/validate/tests/validators_test.py [hunk 2]
```diff
@@ -1025,3 +1025,24 @@ def test_content_release_identifier_exists():
     vm.branch_name = 'master'
     sha1 = vm.get_content_release_identifier()
     assert sha1, 'GIT_SHA1 path in config.yml has been chaged. Fix the demisto-sdk or revert changes in content repo.'
+
+
+@pytest.mark.parametrize('prev_ver, expected', [
+    ('v4.5.0', 'origin/v4.5.0'),
```
Comment (yaakovi) [inline demisto_sdk/commands/validate/tests/validators_test.py:1031]: add a case where the `prev_ver` is a commit hash

**Review by yaakovi — APPROVED** [review]

**Pull request merged**